apis/python/HISTORY.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Changed
 
+- \[[#4126](https://github.com/single-cell-data/TileDB-SOMA/pull/4126)\] [python] at package import time, validate that the expected TileDB version is installed and used. Raises a RuntimeWarning if the condition is not met. This is an attempt to better warn users who have corrupted conda installations.
```

Comment:
```suggestion
- \[[#4126](https://github.com/single-cell-data/TileDB-SOMA/pull/4126)\] [python] At package import time, validate that the expected TileDB version is installed and used. Raises a RuntimeWarning if the condition is not met. This is an attempt to better warn users who have corrupted conda installations.
```

Reply:
I also corrected the HISTORY.md to reflect the `RuntimeError` exception, rather than a warning